integration-tests/smoke/log_poller_test.go

```diff
@@ -36,7 +50,14 @@ func TestLogPollerFewFiltersFixedDepth(t *testing.T) {
 
 	cfg.General.EventsToEmit = eventsToEmit
 
-	logpoller.ExecuteBasicLogPollerTest(t, &cfg)
+	// this was added only to support triggering tests from CI with custom values
+	err := cfg.OverrideFromEnv()
+	if err != nil {
+		t.Errorf("failed to override config from env: %v", err)
+		t.FailNow()
```

Comment:
IIRC this is equivalent to calling `t.Fatalf`:
```suggestion
		t.Fatalf("failed to override config from env: %v", err)
```